tools/analysis_tools/analyze_logs.py

```diff
@@ -48,11 +48,14 @@ def plot_curve(log_dicts, args):
                 legend.append(f'{json_log}_{metric}')
     assert len(legend) == (len(args.json_logs) * len(args.keys))
     metrics = args.keys
+    dataset = args.dataset
 
     num_metrics = len(metrics)
     for i, log_dict in enumerate(log_dicts):
         epochs = list(log_dict.keys())
         for j, metric in enumerate(metrics):
+            if 'mAP' in metric:
+                metric = f'{dataset}/' + metric
```

Comment:
`dataset` seems to be unnecessary. You can remove the prefix in the log_dict's keys so that the logic can be the same as before.